### ISSUE TITLE
Webcams need to include the username for accessibility (even if only visible to screen readers)

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#16048]
**Describe the bug**
In the case when there are lots of webcams displayed or if the area to display webcams is relatively small the webcams shift to a compact display mode where the username is hidden in a menu [so it does not cover the entire webcam stream area]. For screen readers purposes it will be good to include the username in an element tag for the menu trigger.

**To Reproduce**
1. Join a session with several browser tabs
2. Share a webcam in each tab.
3. Shrink the client's area until you see that the webcams switch to compact mode and no longer display the username over each webcam stream.

**Expected behavior**
Screen readers should be able to detect the username without having to open the chevron button menu.

**Actual behavior**
The name is not included.

**Screenshots**
![Screenshot from 2022-11-22 14-48-23](https://user-images.githubusercontent.com/6312397/203409280-7bb09451-a8bd-4c8d-8057-6df8f002b8af.png)
![Screenshot from 2022-11-22 14-48-05](https://user-images.githubusercontent.com/6312397/203409281-58c689f1-0271-46cc-b11b-ffabdc845958.png)

**BBB version:**
BigBlueButton 2.6.0-beta.1
